src/focus.service.ts

```diff
@@ -234,6 +234,7 @@ function isNodeAttached(node: HTMLElement, root: HTMLElement) {
 
 @Injectable()
 export class FocusService {
+  public enableRayTrace = true;
```

Comment:
Wouldn't this be something better set via the ctor or a setter? I always feels weird to configure a service by just setting members.